.babelrc

```diff
@@ -1,12 +1,12 @@
 {
   "presets": [
-    "env",
-    "react"
+    "@babel/preset-env",
+    "@babel/preset-react"
   ],
   "plugins": [
-    "syntax-dynamic-import",
-    "transform-object-rest-spread",
-    "transform-class-properties"
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-class-properties"
```

Comment:
I think we can remove `plugin-proposal-class-properties`, `plugin-proposal-object-rest-spread` and `plugin-syntax-dynamic-import`, they are in `@babel/preset-env`

Reply:
We still need `plugin-proposal-class-properties` and `plugin-syntax-dynamic-import` according to [babel playground](https://babeljs.io/en/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=ATDGBsEMGduAhA9gJwNbAN4CgQgPR4AKyiADgKbIAuAnsAJYB29V9k49AXpcNDY1UgAPHLibRBjUOWJlKtYAF5gAIgBGKVCoDcokBoCujACYAxI6FaJGS4AAoAlEoB8mPbmTkqB5DaoALemgAOnFJaVkKahpdXGAAX113AgBlQVZQMChYYFISKNZyaHcJSAzedPpQSPk6ZXVINXJwAEloAGFERHAdEsrM0ozzKSsbZQAzC1HHNziQT29fBE1gwaqa6PdE0XisegBbUhQqOwByYLxIU4dggPJGO0gXMGtobvJg8EQAc0eHB10oFeVGAalsGG2QMYEmAT2UGGCiLUiSAA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=script&lineWrap=true&presets=env&prettier=false&targets=&version=7.5.5&externalPlugins=%40babel%2Fplugin-proposal-class-properties%407.10.4%2C%40babel%2Fplugin-syntax-dynamic-import%407.8.3).

Reply:
https://github.com/babel/babel/blob/main/packages/babel-preset-env/package.json#L17

Reply:
Do you mean we can remove these from `devDependencies` or from `.babelrc`?

Reply:
And from `devDependencies` and from `.babelrc`

Reply:
I've pushed another commit removing all those plugins, as you can see, both the Vercel deployment and [CI failed](https://travis-ci.org/github/webpack/webpack.js.org/jobs/725574269#L255).

Regarding the `devDependencies`, since we're using those plugins in `.babelrc`, wouldn't it be a good idea to define them explicitly instead of using those included in `@babel/preset-env`?